docs/install.md

```diff
@@ -5,11 +5,41 @@
 
 * **Python**: Required. Model Server for Apache MXNet (MMS) works with Python 2 or 3.  When installing MMS, we recommend that you use a Python and Conda environment to avoid conflicts with your other Apache MXNet or Open Neural Network Exchange (ONNX) installations.
 
-* **protoc**: Optional. If you plan to use ONNX features, you need to install the [protobuf compiler](https://github.com/onnx/onnx#installation). Install it *before* installing MMS.
+* **java 8**: Required. MMS use java to serve HTTP requests. You must install java 8 (or later) and make sure java is on available in $PATH environment variable *before* installing MMS. If you have multiple java installed, you can use $JAVA_HOME environment vairable to control which java to use.
+
+For ubuntu:
+```bash
+sudo apt-get install openjdk-8-jre-headless
+```
+
+For centos
+```bash
+sudo yum install java-1.8.0-openjdk
+```
+
+For Mac:
+```bash
+brew tap caskroom/versions
+brew update
+brew cask install java8
+```
+
+* **MxNet**: Recommended. MMS won't install `mxnet` by default. MxNet is required for most of examples in this project. MMS won't install mxnet engine by default, you can install mxnet-mkl or mxnet-cu90mkl based on your need. And you can also choose specific version of mxnet if you want.
```

Comment:
I think we do install. This needs to be rewritten to showcase the extras_require... Might be better to show the options. 
We should ask customers to do pip install if they have requirements for specific version of MXNet or other frameworks